tock/tock/templates/employees/user_form.html

```diff
@@ -1,11 +1,10 @@
 {% extends "base.html" %}
 {% block navigation %}
-  {% include "_navigation.html" %}
+{% include "_navigation.html" %}
 {% endblock %}
 
 {% block content %}
-  <h1>Let's Tock about {{ username }}!</h1>
-  {% include 'employees/user_utilization.html' %}
-  {% include 'employees/user_recent_tocks.html' %}
-{% endblock %}
-
+<h1>Let's Tock about {{ username }}!</h1>
+{% include 'employees/user_utilization.html' %}
+{% include 'employees/user_recent_tocks.html' %}
+{% endblock %}
```

Comment:
Not looking for any change here, but does it bother anyone else that we don't indent within blocks? Should we have a convention to start doing so, or should I just let it go :frozen:

Reply:
Sorry, this is probably my bad -- VSCode sometimes automatically un-indents things and it drives me crazy but I can't figure out how to turn it off 😬

Reply:
possibly this setting in VSCode editor settings?  
![image](https://user-images.githubusercontent.com/22898229/79480147-58a11480-7fd3-11ea-80bf-53e93526769c.png)